package.json

```
@@ -20,6 +20,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "lint": "eslint .",
+    "lint:fix": "eslint --fix test/* src/*",
```

The sentry js package use only `fix` for lint --fix. I would keep the naming for consistency and dex between packages.

```suggestion
    "fix": "eslint --fix test/* src/*",
```